app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError(
```

Comment:
In this exercise we recommend to keep code simple. Instead of raising exception here you can structure conditionals in a way that if conditions are met you continue with the rest of code, otherwise nothing will be executed and the program will exit. This will save you couple of lines of code and you may write less instructions

Reply:
And this?